alicloud/resource_alicloud_ess_scalinggroup.go

```diff
@@ -77,6 +77,10 @@ func resourceAlicloudEssScalingGroup() *schema.Resource {
 				ForceNew: true,
 				MinItems: 0,
 			},
+			"multi_az_policy": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
I think this new field 'multi_az_policy' needs a ValidateFunc and Default

Reply:
Agreed. I'll make the change over the weekend.